genesis/engine/solvers/rigid/collider_decomp.py

```diff
@@ -322,7 +354,21 @@ def detection(self) -> None:
         # timer = create_timer(name="69477ab0-5e75-47cb-a4a5-d4eebd9336ca", level=3, ti_sync=True, skip_first_call=True)
         self._func_update_aabbs()
         # timer.stamp("func_update_aabbs")
-        self._func_broad_phase()
+        self._func_broad_phase(
+            self._solver.links_state,
+            self._solver.links_info,
+            self._solver.geoms_state,
+            self._solver.geoms_info,
+            self._solver._rigid_global_info,
+            self._solver._static_rigid_sim_config,
+            self.collision_pair_validity,
+            self.broad_buffer,
+            self.broad_first_time,
+            self.n_broad_pairs,
+            self.broad_collision_pairs,
+            self._max_collision_pairs,
```

Comment:
Can we store this in a ti.field? If it's a constant, the kernel will need to be recompiled when its value changes.